test/api-console-request.test.js

```diff
@@ -0,0 +1,144 @@
+import {fixture, assert, html, nextFrame} from '@open-wc/testing';
+import * as sinon from 'sinon';
+import {AmfLoader, ApiDescribe} from './amf-loader.js';
+import '../api-console.js';
+import {
+  documentationTryItButton,
+  navigationSelectEndpointMethod,
+  requestBodySection,
+  requestHeadersSection,
+  requestQueryParamSection, requestSendButton,
+  requestUrlSection
+} from './testHelper.js';
+
+/** @typedef {import('..').ApiConsole} ApiConsole */
+
+describe('API Console request', () => {
+  /**
+   * @returns {Promise<ApiConsole>}
+   */
+  async function amfFixture(amf) {
+    return (fixture(html`
+        <api-console .amf="${amf}"></api-console>
+    `));
+  }
+
+  const testApi = 'test-api';
+
+  [
+    new ApiDescribe('Regular model'),
+    new ApiDescribe('Compact model', true)
+  ].forEach(({label, compact}) => {
+    describe(label, () => {
+      let element;
+      let amf;
+      let spy;
+
+      before(async () => {
+        amf = await AmfLoader.load({compact, fileName: testApi});
+      });
+
+      beforeEach(async () => {
+        element = await amfFixture(amf);
+      });
+
+      describe('Sections', () => {
+        beforeEach(async () => {
+          await navigationSelectEndpointMethod(element, '/test-headers', 'post');
+          await nextFrame();
+          await nextFrame();
```

Comment:
Maybe use `await aTimeout(50)` or something to avoid using so many `nextFrame()`? Is there any drawback/benefit to using one or the other?

Reply:
Or maybe we could have a custom `nextFrames(n)` where we call `nextFrame()` n times. Just a thought. Not really super necessary